Makefile

```diff
@@ -26,7 +35,7 @@ install:
 	cp runc /usr/local/bin/runc
 
 clean:
-	rm runc
+	rm runc || true
```

Comment:
On Mon, Sep 07, 2015 at 07:56:45PM -0700, Lai Jiangshan wrote:
> > -	rm runc
> > +	rm runc || true
> 
> I think the intention of the author is that " if the file runc is
> non-existed, it shouldn't fail".

rm(1) has:

       -f, --force
              ignore nonexistent files and arguments, never prompt

so there's no need for anything fancier than ‘rm -f runc’.